integration-tests/utils/__init__.py

```diff
@@ -12,6 +12,14 @@ def yggdrasil_service_is_active():
         return False
 
 
+def show_yggdrasil_service_status():
+    """This method return status of yggdrasil service.
+    :return: status
+    """
+    status = sh.systemctl("status yggdrasil".split()).strip()
```

Comment:
the `sh` library can throw additional tracebacks depending on the systemctl's exit code, which it likely will if this is ever run (this code only happens in exceptions).  To make debugging cleaner would this benefit from either exception handling, or even using `subprocess` - maybe even pytest_client_tools'  `logged_run` ?

Reply:
make sense to use existing logged_run.Thanks for feedback!

Reply:
well no, `logged_run()` is currently NOT public API (even if it isn't named with `_`)